src/app/collections/collection.directive.js

```diff
@@ -45,7 +45,6 @@
         vm.triggerClearFilters = triggerClearFilters;
         //vm.triggerRestoreState = triggerRestoreState;
         vm.triggerSearch = triggerSearch;
-
         activate();
```

Comment:
All of the other controllers have a space between the list of "public" methods and the "activate()" call